internal/config.go

```diff
@@ -174,22 +175,32 @@ func isAllowedSetting(setting string, AllowedSettings map[string]bool) (exists b
 	return
 }
 
+// GetSetting extract setting by key if key is set, return empty string otherwise
 func GetSetting(key string) (value string, ok bool) {
-	if viper.IsSet(key) {
-		return viper.GetString(key), true
+	return GetSettingFrom(viper.GetViper(), key)
+}
+
+// GetSettingFrom extract setting by key from config if key is set, return empty string otherwise
+func GetSettingFrom(config *viper.Viper, key string) (value string, ok bool) {
+	if config.IsSet(key) {
+		return config.GetString(key), true
 	}
 	return "", false
 }
 
 func getWaleCompatibleSetting(key string) (value string, exists bool) {
+	return getWaleCompatibleSettingFrom(key, viper.GetViper())
```

Comment:
GetSettingFrom has the same arguments but in another order. Use the same order in both places.

Reply:
unexpectedly GetSettingFrom method was not needed